### PR TITLE
Fixes vehicle bug which caused you to keep special actions when you were forcemoved out of them through weird means (Admin intervention, special items) 

### LINKED ISSUE #40202: Stealth box and Clown Car have strange interactions.
Issue reported from  Round ID: 93683 (/tg/Station Sybil [ENGLISH] [US-EAST] [100% LAG FREE])

Reporting client version: 512

I got scooped up in a clown car and activated my stealth implant. This caused me to get inside the box outside the car and go about my merry way. Additionally, I could hit the "eject" button to resist out of the car, which caused me to teleport TO the car and eject everyone inside it.  Even later still I used my stealth implant again which teleported me back to the car, but not inside it.

### PR DESCRIPTION
**Pull request by ghost**
:cl: Floyd / Qustinnus
fix: Fixes vehicle bug which caused you to keep special actions when you were forcemoved out of them through weird means (Admin intervention, special items) 
/:cl:

fixes #40202

### CODE REVIEW

**Comment by SpaceManiac** [comment]
Not sure if this is the right fix. Does teleporting outside a clown car let you keep the "exit clown car" action to teleport back?

**Comment by Qustinnus** [comment]
yes. the stealth implant force moves you to the drop_location

**Comment by SpaceManiac** [comment]
I mean teleporting by some other means, like a scroll or adminnery

**Review by SpaceManiac — DISMISSED** [review]

File: code/modules/vehicles/sealed.dm [hunk 1]
```diff
@@ -19,6 +19,11 @@
 		mob_try_enter(M)
 	return ..()
 
+/obj/vehicle/sealed/Exited(atom/movable/AM, atom/newLoc)
+	. = ..()
+	if(isliving(AM))
```
Comment (ShizCalev) [inline code/modules/vehicles/sealed.dm]: set this to ismob

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Review by ShizCalev — APPROVED** [review]

**Pull request merged**